packages/oxlint-config/src/index.test.ts

```
@@ -310,9 +316,13 @@ describe("oxlint-config", () => {
           "unicode-bom": ["error", "never"],
           "use-isnan": "error",
           "valid-typeof": "error",
-        },
+        }),
       });
 
+      const reactDoctorRules = getRulesWithPrefix(frontend.rules ?? {}, "react-doctor/");
```

ESLint

[error] Avoid having conditionals in tests

(jest/no-conditional-in-test)